website.i2pd.i2pd.json

```diff
@@ -0,0 +1,107 @@
+{
+    "app-id": "website.i2pd.i2pd",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "i2pd_qt_wrapper",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri"],
+    "cleanup": [
+        "/cache",
+        "/share/doc", "/share/man", "/bin/openssl", "/bin/upnpc", "/bin/c_rehash", "/bin/external-ip",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --with-libraries=date_time,filesystem,program_options,system",
+                "./b2 -j $FLATPAK_BUILDER_N_JOBS install --prefix=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+                    "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+                }
+            ]
+        },
+        {
+            "name": "openssl",
```

Comment:
Could you perhaps put a comment here pointing out that once the KDE runtime is based on FDO 18.08 (maybe KDE 5.12? dunno). This should be removed.

Reply:
It currently uses openssl v1.1.1 for the best performance. Sure, when upstream SDK will support 1.1.1, it will be removed.